lib/manageiq/messaging/stomp/client.rb

```diff
@@ -61,7 +61,7 @@ class Client < ManageIQ::Messaging::Client
         attr_reader :stomp_client
 
         def initialize(options)
-          host = options.slice(:host, :port)
+          host = options.slice(:host, :port, :ssl)
```

Comment:
NOTE stomp accepts an ssl option in the host parameter: https://github.com/stompgem/stomp
```
hash = {
  hosts: [
    # First connect is to remotehost1
    { login: 'login1', passcode: 'passcode1', host: 'remotehost1', port: 61_612, ssl: true },
    # First failover connect is to remotehost2
    { login: 'login2', passcode: 'passcode2', host: 'remotehost2', port: 61_613, ssl: false }
  ],
```